packages/test-utils/src/find.js

```diff
@@ -24,6 +25,10 @@ export default function find (
     throwError('cannot find a Vue instance on a DOM node. The node you are calling find on does not exist in the VDom. Are you adding the node as innerHTML?')
   }
 
+  if (!vm && selectorType === REF_SELECTOR) {
```

Comment:
Moved validation before search code. I think code is more readable when validation is separated from rest logic